src/components/TableOfContents.js

```diff
@@ -345,7 +342,7 @@ const TableOfContents = ({
     )
   }
 
-  const shouldShowEditButtom = editPath && intl.locale === "en"
+  const shouldShowEditButtom = editPath && editPath.length > 0
```

Comment:
Could probably trim this further and force it to a boolean
```suggestion
  const shouldShowEditButtom = !!editPath
```

Reply:
Good point! Changed.